pandas/tests/extension/base/ops.py

```diff
@@ -158,13 +156,3 @@ def test_compare_array(self, data, all_compare_operators):
         s = pd.Series(data)
         other = pd.Series([data[0]] * len(data))
         self._compare_other(s, data, op_name, other)
-
-    def test_direct_arith_with_series_returns_not_implemented(self, data):
```

Comment:
Undo this change.

Reply:
> Same as the other comment. The whole PR is intended to break this test since we implement __eq__ and __ne__.

The reason this needs to be kept is that, although `__eq__` is a required method on an ExtensionArray now, we still require it to return NotImplemented when `other` is a Series or Index.